scripts/load_safe_deployments.py

```diff
@@ -0,0 +1,75 @@
+import json
+
+import requests
+
+safe_deployments_base_url = (
+    "https://raw.githubusercontent.com/safe-global/safe-deployments/main/src/assets"
+)
+safe_deployments_path = "../gnosis/safe/safe-deployments/safe-deployments.json"
```

Comment:
Maybe rename to `output_file` or add a comment pointing out that this will be the output file